src/_cffi_src/openssl/err.py

```diff
@@ -193,6 +190,16 @@
 static const int SSL_AD_INTERNAL_ERROR;
 static const int SSL_AD_ACCESS_DENIED;
 static const int SSL_AD_HANDSHAKE_FAILURE;
+
+static const int ASN1_F_B64_READ_ASN1;
+static const int ASN1_F_B64_WRITE_ASN1;
+static const int ASN1_F_SMIME_READ_ASN1;
+static const int ASN1_F_SMIME_TEXT;
+static const int ASN1_R_NO_CONTENT_TYPE;
+static const int ASN1_R_NO_MULTIPART_BODY_FAILURE;
+static const int ASN1_R_NO_MULTIPART_BOUNDARY;
+static const int EVP_F_CAMELLIA_INIT_KEY;
+static const int EVP_R_CAMELLIA_KEY_SETUP_FAILED;
```

Comment:
Actually, I think there's existing groupings of consts that these could be added to.